test/sequential/bootstrap.cancelBootstrap.test.ts

```diff
@@ -52,9 +52,7 @@ beforeAll(async () => {
   keyring = new Keyring({ type: "sr25519" });
 
   sudo = new User(keyring, sudoUserName);
-});
 
-beforeEach(async () => {
```

Comment:
Are you sure about this?  It seems that this will only create one asset for all the tests. And the precondition of the check-bootstrap finished won't be done.